src/test/resources/exampleFolders/sampleReports/AddressesReport.jrxml

```diff
@@ -1,137 +1,136 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Maybe for reasons of clarity and comprehensibility (git history) it's better to revert those formatting changes?

Reply:
Indeed! (But JasperSoft Studio thinks it knows better :-) 
I reverted the reformatting and pushed a new version of the file that only includes compatibility changes.